### PR TITLE
Chore/aws cache

### PR DESCRIPTION
**Pull request by briri**
Was able to setup SessionManager and log onto one of the ECS containers. From there I was able to connect to the Redis cache but had to use TLS.

This PR contains a modification to the config for the cache to use TLS when not in development mode. I also added a couple of debug lines (which I will remove once it's working).

### CODE REVIEW

File: src/config/cacheConfig.ts [hunk 1]
```diff
@@ -6,8 +6,10 @@ dotenv.config();
...
-  port: Number.parseInt(process.env.CACHE_PORT),
-  connectTimeout: Number.parseInt(process.env.CACHE_CONNECT_TIMEOUT) || 30000, // 30 seconds
-};
+const host = process.env.CACHE_HOST;
+const port = Number.parseInt(process.env.CACHE_PORT);
+const connectTimeout = Number.parseInt(process.env.CACHE_CONNECT_TIMEOUT) || 30000; // 30 seconds
+
+export const cacheConfig = { host, port, connectTimeout };
+
+export const cacheTLS = `rediss://${host}:${port}`;
```
Comment (jupiter007) [inline src/config/cacheConfig.ts:15]: L15: Just checking that it's "rediss" with two "s"es.

Also, is it a good idea to mention that we use TLS for cache when not in development mode somewhere in the README?

Comment (jupiter007) [reply src/config/cacheConfig.ts:15]: Oh, and also, should you update the CHANGELOG?

Comment (briri) [reply src/config/cacheConfig.ts:15]: Yeah, was going to hold off on all that until I know these changes will work

**Review by jupiter007 — APPROVED** [review]

**Comment by jupiter007** [comment]
Ok, approved the PR. Here's hoping it works :)

**Pull request merged**